maa-cli/src/config/mod.rs

```diff
@@ -212,6 +215,81 @@ pub fn convert(file: &Path, out: Option<&Path>, ft: Option<Filetype>) -> Result<
     }
 }
 
+pub fn cleanup(inquire: bool) -> Result<(), io::Error> {
+    let state = dirs::state().to_path_buf();
+    let debug = dirs::log().to_path_buf();
```

Comment:
There is no need to convert these `&Path` to `PathBuf`, which will be done in `join`.